variantlib/models/variant.py

```diff
@@ -111,12 +112,12 @@ class VariantProperty(VariantFeature):
         }
     )
 
-    @cached_property
+    @property
     def property_hash(self) -> int:
         # __class__ is being added to guarantee the hash to be specific to this class
         return hash((self.__class__, self.namespace, self.feature, self.value))
 
-    @cached_property
+    @property
```

Comment:
I don't think this is a good use of property. The returned object is not a property (i.e. something belonging to) of the original `VariantProperty` but rather a completely new, independent object. The usual way of constructing new objects in Python is via method and not property.

Also, it would be more consistent to follow the naming scheme of `to_*` since it's converting another type.

Reply:
What about something like `pathlib.Path().parent` ? It does also return a new `Path()`

Like `.parent_feature` - I don't like much `feature_object` taking ideas if you have smthg in mind.

If you really want it as a function we can - this API is pretty much internal-only.

Reply:
Yeah, that's a fair point. The stdlib is already inconsistent, so I guess it's fine either way. (but then, `to_str()` could be a property too, and that would be mightily confusing.)